doc/classes/OS.xml

```diff
@@ -638,25 +691,26 @@
 			<return type="int" enum="Error" />
 			<param index="0" name="name" type="String" />
 			<description>
-				Sets the name of the current thread.
+				Assigns the given name to the current thread. Returns [constant ERR_UNAVAILABLE] if unavailable on the current platform.
 			</description>
 		</method>
 		<method name="set_use_file_access_save_and_swap">
 			<return type="void" />
 			<param index="0" name="enabled" type="bool" />
 			<description>
-				Enables backup saves if [param enabled] is [code]true[/code].
+				If [param enabled] is [code]true[/code], when opening a file for writing, a temporary file is used in its place. When closed, it is automatically applied to the target file.
+				This can useful when files may be opened by other applications, such as antiviruses, text editors, or even the Godot Editor itself.
```

Comment:
```suggestion
				This can useful when files may be opened by other applications, such as antiviruses, text editors, or even the Godot editor itself.
```